deploy/local/vars_win.bat

```diff
@@ -52,6 +52,9 @@ if exist "%DAAL_UP_OLD%\redist" (
 )
 set "CLASSPATH=%DAAL%\lib\onedal.jar;%CLASSPATH%"
 endlocal& ^
+set DAL_MAJOR_BINARY=1& ^
+set DAL_MINOR_BINARY=0& ^
+set DALROOT=%DAALROOT%& ^
```

Comment:
I see section `:GoodArgs` at line 43, don't we need to add `DAL_MAJOR`/`DAL_MINOR` there?

Reply:
Thins within setlocal context is get used for different data manipulations and calculations but actual values are set after endlocal section.  As far we have no processing for now - it's sufficient to specify them at the end. 

but as this have to be moved to version parsing - this logic will be moved to setlocal context and GoodArgs point